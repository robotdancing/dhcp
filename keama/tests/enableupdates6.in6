# ddns-updates (aka enable-updates)

# authoritative is mandatory
authoritative;

ddns-updates on;

# embedded
class "foo" {
    ddns-updates off;
}
