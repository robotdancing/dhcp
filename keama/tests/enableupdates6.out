{
  # ddns-updates (aka enable-updates)
  # authoritative is mandatory
  "Dhcp6": {
    "dhcp-ddns": {
      "enable-updates": true
    },
    "client-classes": [
      # embedded
      {
        "name": "foo"
//      /// Only global enable-updates is supported
//      "enable-updates": false
      }
    ]
  }
}
